torch/quantization/fx/prepare.py

```diff
@@ -924,10 +955,11 @@ def insert_observers_for_model(
                         node_name_to_target_dtype)
 
                     is_last_node_of_pattern = root_node is node
+                    # TODO: add is_like_copy_node method in quantizehandler
```

Comment:
maybe we can do this in this PR?  `qhandler.num_tensor_args` is currently not used outside of `quantization_patterns.py`, would be great to keep it private.

Reply:
OK, sure